addons/mail/static/src/rtc/call_invitations.xml

```diff
@@ -5,7 +5,7 @@
         <div class="o-mail-CallInvitations position-absolute top-0 end-0 d-flex flex-column p-2">
             <t t-if="rtc.ringingThreads">
                 <t t-foreach="rtc.ringingThreads" t-as="threadLocalId" t-key="threadLocalId">
-                    <t t-set="thread" t-value="store.threads[threadLocalId]"/>
+                    <t t-set="thread" t-value="discussStore.threads[threadLocalId]"/>
```

Comment:
Same comment: accessing `discuss` from outside its folder is not correct.

Reply:
same comment: I'll move the whole rtc folder to discuss.